.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/phobos
+    docker:
+      - image: circleci/node:4.8.2
+    parallelism: 1
+    steps:
+      - checkout
+      - run:
+          command: ./.circleci/run.sh install-deps
+          name: Install DMD
+      - run:
+          command: ./.circleci/run.sh setup-repos
+          name: Clone DMD & DRuntime
+      - run:
+          command: sudo apt-get update && sudo apt-get install -y gdb
+          name: Install gdb
+      - run:
+          command: ./.circleci/run.sh style_lint
+          name: Run code style & linter
```

Comment:
Currently I get the following:

```
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
core.exception.OutOfMemoryError@src/core/exception.d(702): Memory allocation failed
----------------
[Inferior 1 (process 2163) exited with code 01]
No stack.
```

See also: https://circleci.com/gh/wilzbach/phobos/13
Sadly `gdb` doesn't forward the exit code, [it's apparently a well-known bug](https://sourceware.org/bugzilla/show_bug.cgi?id=13000).